src/consts/resources.py

```diff
@@ -1,12 +1,12 @@
 from consts import DISK_SIZE_100GB, GB
 
-DEFAULT_WORKER_MEMORY: int = 8892
-DEFAULT_MASTER_MEMORY: int = 16984
+DEFAULT_WORKER_MEMORY: int = 8 * 1024  # mebibytes
```

Comment:
Why not declare `MiB = 1024` as same as we did on `GB`?

Reply:
That will make all the comments redundant

Reply:
``MiB = 1024`` seems incorrect, because 1024 is actually K

Reply:
I initially did ``(8 * GiB) // MiB`` or something alike, not sure what's more clear of the two options

Reply:
Maybe something like `DEFAULT_WORKER_MEMORY: int = 3 * MiB_UNITS` where `MiB_UNITS=1024`?

Reply:
Done!